src/adminApi.js

```diff
@@ -20,7 +20,7 @@ export default ({host, https, ignoreConsumers, cache}) => {
 function createApi({ router, getPaginatedJson, ignoreConsumers }) {
     return {
         router,
-        fetchGlobalPlugins: () => getPaginatedJson(router({name: 'plugins'})),
+        fetchAllPlugins: () => getPaginatedJson(router({name: 'plugins'})),
```

Comment:
This was just misnamed. `/plugins` returns all configured plugins, including those that target routes, services, APIs, and consumers.